examples/DataCollection.cpp

```diff
@@ -46,7 +46,7 @@ void loop() {
     for (size_t i = 0; i < numSensors; i++)
     {
         Sensor* sensor = sensors[i];
-        if (sensor->ready())
+        if (sensor->healthCheck() && sensor->ready())
```

Comment:
Idk if this is in the real DC yet but having an if health check as a read condition breaks the car.

The ECU needs to detect if the throttles or brake get unplugged during running so we always need to hand off the value even if it's 0 so the ECU can make the decision to shut the car off.